chat-server/pom.xml

```diff
@@ -2,31 +2,24 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.aionemu</groupId>
+	<parent>
+		<groupId>com.aionemu</groupId>
+		<artifactId>aion-server</artifactId>
+		<version>4.8.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
 	<artifactId>chat-server</artifactId>
-	<version>4.8-SNAPSHOT</version>
 	<name>Aion Chat Server</name>
 	<packaging>jar</packaging>
+
 	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.test.skip>true</maven.test.skip>
 	</properties>
+
 	<build>
 		<sourceDirectory>src</sourceDirectory>
 		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.13.0</version>
-				<configuration>
-					<release>21</release>
-					<compilerArgs>
-						<arg>-Xlint:all,-preview,-this-escape</arg>
-					</compilerArgs>
-					<showWarnings>true</showWarnings>
-					<debug>true</debug>
```

Comment:
Not sure if removing these parameters in all projects is a good or bad thing. Some of the warnings have been helpful in the past but I can't remember new valid ones in more recent years that I had to fix 🤔

Reply:
funny enough I couldn't build a single module with these parameters. I'll double-check.